### PR TITLE
Update tests and CI for Ruby 2.2.0

### PR DESCRIPTION
**Pull request by Lordnibbler**
## Status
**READY**

## Description
Update tests and CI configuration for Ruby 2.2.0

## Todos
- [x] Update tests (thanks @phlipper!)
- [x] Update CI configuration

### CODE REVIEW

File: test/response_test.rb [hunk 1]
```diff
@@ -195,15 +195,23 @@ class RubySamlTest < Minitest::Test
...
 
-      it "optionally allow for clock drift" do
+      it "optionally allows for clock drift" do
         # The NotBefore condition in the document is 2011-06-14T18:21:01.516Z
-        Time.stubs(:now).returns(Time.parse("2011-06-14T18:21:01Z"))
-        response = OneLogin::RubySaml::Response.new(response_document_5, :allowed_clock_drift => 0.515)
-        assert !response.send(:validate_conditions, true)
+        Timecop.freeze(Time.parse("2011-06-14T18:21:01Z")) do
+          response = OneLogin::RubySaml::Response.new(
+              response_document_5,
```
Comment (phlipper) [inline test/response_test.rb]: Nitpick, but the whitespace is off a bit :) Same below

**Comment by pitbulk** [comment]
;)

**Comment by phlipper** [comment]
:+1:

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #
 # Please keep this file alphabetized and organized
 #
-source 'http://rubygems.org'
+source 'https://rubygems.org'
```
Comment (Umofomia) [inline Gemfile]: If you're making this change here, you might want to make it in `gemfiles/nokogiri-1.5.gemfile` as well.

Comment (Lordnibbler) [inline Gemfile]: done, good call

**Pull request merged**